src/scripts/public/dom-loader.js

```diff
@@ -7,11 +7,14 @@ const domReady = () => {
 	 */
 	setTimeout(function() {
```

Comment:
Just a note, we prefer the use of arrow functions on the team and try to stay consistent with those.